sympy/sets/sets.py

```diff
@@ -1628,6 +1628,10 @@ class FiniteSet(Set, EvalfMixin):
     >>> 3 in FiniteSet(1, 2, 3, 4)
     True
 
+    >>> members = [1, 2, 3, 4]
+    >>> FiniteSet(*members)
```

Comment:
I understand your point. This doctest aims to provide an example for the use case I mention in the second part of my previous comment. So, it may be a common Python thing and one that a SymPy user is expected to know, but there is a possibility they may not.